guides/common/modules/proc_troubleshooting-permission-issues.adoc

```diff
@@ -0,0 +1,34 @@
+[id="troubleshooting-permission-issues"]
+= Troubleshooting permission issues
+
+{Project} upgrades perform pre-upgrade checks.
+If your {Project} upgrade fails due to permission issues, you can identify and remedy them on your {ProjectServer}.
+
+You can perform this procedure if you receive an error similar to the following one:
+
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+2024-01-29T20:50:09 [W|app|] Could not create role 'Ansible Roles Manager': ERF73-0602 [Foreman::PermissionMissingException]: some permissions were not found:
+----
```

Comment:
I'd like to find a way to avoid self-referential wording ("This procedure").

Also, it's not the upgrade that fails, is it? I'd like to assume the error is displayed in the pre-upgrade check phase, before the upgrade itself even starts.

I tried shuffling words around a bit and would suggest something like this. What do you think?

```suggestion
If the pre-upgrade check discovers permission issues, it fails with an error similar to the following one:

[options="nowrap", subs="verbatim,quotes,attributes"]
----
2024-01-29T20:50:09 [W|app|] Could not create role 'Ansible Roles Manager': ERF73-0602 [Foreman::PermissionMissingException]: some permissions were not found:
----

If you see an error like this on your {ProjectServer}, identify and remedy the permission issues.
```

Reply:
Yes, only the pre-upgrade check fails, but that means as a user, the upgrade process fails.

Applied.